app/views/layouts/application.html.erb

```diff
@@ -10,7 +10,8 @@
 
   <body>
     <% if signed_in? %>
-      Signed in as: <%= current_user.email %>
+      Signed in as: <%= current_user.name %>
+      <%= image_tag current_user.avatar_url, size: '70' if current_user.avatar? %>
```

Comment:
We could extract this to a constant on the user model, e.g. `User::AVATAR_SIZES[:small]`